g419-liner2-cli/build.gradle

```diff
@@ -41,4 +41,5 @@ dependencies {
     compile 'commons-cli:commons-cli:1.2'
     compile 'commons-io:commons-io:2.4'
     compile files('../lib/PolemJava.jar')
+    compile files('../lib/fasttext.jar')
```

Comment:
Co to za zależność? Gdzie są kody dla tego jar-a?

Reply:
Liner2/lib/